From b0ea33ab7d2688c2015db23d8b11e67148ab67a8 Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Fri, 6 Sep 2019 08:29:50 +0000
Subject: ath79-cpe510v2v3-backport-mk

Fixes: 853e4dd3 ("ipqx0xx: add Generic subtarget")
Ref: https://forum.openwrt.org/t/ipq40xx-snapshot-not-updated-since-22nd-august/44126
Signed-off-by: Yousong Zhou <yszhou4tech@gmail.com>
---
 package/boot/uboot-fritz4040/Makefile | 1 +
 1 file changed, 1 insertion(+)

commit b94dc4646a8fc129a374608594e6a5dbd823f215
Author: Sven Roederer <devel-sven@geroedel.de>
Date:   Sat Aug 10 17:13:57 2019 +0200

    ath79: revert DEVICE_MODEL and olki-loader for 19.07 (CPE510v2v3)

diff --git a/target/linux/ath79/image/generic-tp-link.mk b/target/linux/ath79/image/generic-tp-link.mk
index 0caaaaa55b..7484c66637 100644
--- a/target/linux/ath79/image/generic-tp-link.mk
+++ b/target/linux/ath79/image/generic-tp-link.mk
@@ -150,15 +150,10 @@ define Device/tplink_cpe510-v2
   $(Device/tplink-safeloader)
   ATH_SOC := ar9344
   IMAGE_SIZE := 7680k
-  DEVICE_MODEL := CPE510
-  DEVICE_VARIANT := v2
+  DEVICE_TITLE := TP-Link CPE510 v2
   DEVICE_PACKAGES := rssileds
   TPLINK_BOARD_ID := CPE510V2
   LOADER_TYPE := elf
-  LOADER_FLASH_OFFS := 0x43000 
-  COMPILE := loader-$(1).elf 
-  COMPILE/loader-$(1).elf := loader-okli-compile 
-  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma -M 0x4f4b4c49 | loader-okli $(1) 12288
   SUPPORTED_DEVICES += cpe510-v2
 endef
 TARGET_DEVICES += tplink_cpe510-v2
@@ -167,15 +162,10 @@ define Device/tplink_cpe510-v3
   $(Device/tplink-safeloader)
   ATH_SOC := ar9344
   IMAGE_SIZE := 7680k
-  DEVICE_MODEL := CPE510
-  DEVICE_VARIANT := v3
+  DEVICE_TITLE := TP-Link CPE510 v3
   DEVICE_PACKAGES := rssileds
   TPLINK_BOARD_ID := CPE510V3
   LOADER_TYPE := elf
-  LOADER_FLASH_OFFS := 0x43000 
-  COMPILE := loader-$(1).elf 
-  COMPILE/loader-$(1).elf := loader-okli-compile 
-  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma -M 0x4f4b4c49 | loader-okli $(1) 12288
   SUPPORTED_DEVICES += cpe510-v3
 endef
 TARGET_DEVICES += tplink_cpe510-v3
-- 
2.20.1

